### PR TITLE
[Rust] Update standalone config

### PR DESCRIPTION
**Pull request by apepkuss**
In this PR, the config related to the `standalone` mode has been updated for the coming release.

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by apepkuss** [comment]
@hydai `wasmedge` failed to build on `Fedora 37`. Please refer to [the details](https://github.com/WasmEdge/WasmEdge/actions/runs/4826819865/jobs/8598999543?pr=2450)

File: bindings/rust/wasmedge-sys/README.md [hunk 1]
```diff
@@ -6,21 +6,6 @@ For developers, it is recommended that the APIs in `wasmedge-sys` are used to co
 
 * Notice that [wasmedge-sys](https://crates.io/crates/wasmedge-sys) requires **Rust v1.66 or above** in the **stable** channel.
 
-## Versioning Table
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/README.md:9]: Let's maintain this table. Why do you remove it?

Comment (apepkuss) [reply bindings/rust/wasmedge-sys/README.md:9]: The versioning table is still there. Just move it into a new section of this file. Thanks!

**Review by hydai — CHANGES_REQUESTED** [review]

File: bindings/rust/wasmedge-sys/README.md (hunk 1 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sys/README.md:9]: We moved the versioning table into the `Build` section in the previous PR, and in that PR we forgot to remove the old one. Thanks.

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**